### PR TITLE
time block in logger only

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: src/logger.rs [hunk 1]
```diff
@@ -30,6 +31,15 @@ pub async fn logger_manager(
 ) -> Result<(), Box<dyn Error + Send + Sync>> {
     let mut writer: Option<BufWriter<File>> = None;
 
+    // time is wrong for a while upon boot.  hold on until it is OK
+    while !SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .is_ok_and(|time| time > Duration::from_millis(1730247194876))
```
Comment (caiodasilva2005) [inline src/logger.rs:37]: is this timeout in ms arbitrary?

File: src/mqtt_handler.rs [hunk 2]
```diff
@@ -79,9 +79,8 @@ impl MqttProcessor {
         mqtt_opts
             .set_keep_alive(Duration::from_secs(20))
             .set_clean_start(false)
-            .set_connection_timeout(3)
-            //       .set_session_expiry_interval(Some(u32::MAX))
-            .set_topic_alias_max(Some(600));
```
Comment (caiodasilva2005) [inline src/mqtt_handler.rs:84]: What is the problem with topic aliases?

File: src/logger.rs (hunk 1 above)
Comment (jr1221) [reply src/logger.rs:37]: this is like the year 2000 or something.  its to check for a "sane" time.

File: src/mqtt_handler.rs (hunk 2 above)
Comment (jr1221) [reply src/mqtt_handler.rs:84]: ah the way rumqttc does them is fucked.  It is causing mosquitto to give warnings and nanoMQ to outright fail, and it is useless since Scylla requires topic aliases are disabled.
